src/main/java/frc/robot/subsystems/intake/IntakeIOTalonFX.java

```diff
@@ -0,0 +1,200 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix6.BaseStatusSignal;
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.configs.CurrentLimitsConfigs;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.VelocityTorqueCurrentFOC;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.lib.team3061.RobotConfig;
+import frc.lib.team6328.util.TunableNumber;
+
+public class IntakeIOTalonFX implements IntakeIO {
+  private TalonFX rightRollerMotor;
+  private TalonFX leftRollerMotor;
+  private TalonFX drumMotor;
+  private final DigitalInput rightRollerIRSensor;
+  private final DigitalInput leftRollerIRSensor;
+  private final DigitalInput drumIRSensor;
+
+  private double rightRollerRequestedVelocity;
```

Comment:
Delete these three instance variables as we can get the reference velocity from the device.